vividus-facade-jira/src/main/resources/org/vividus/jira/spring.xml

```diff
@@ -6,16 +6,16 @@
        default-lazy-init="true">
 
     <bean id="caching-credentials" class="org.vividus.http.context.CachingCredentialsHttpContextFactory" >
-        <constructor-arg index="0" value="${jira.username}"/>
-        <constructor-arg index="1" value="${jira.password}"/>
-        <constructor-arg index="2" value="${jira.endpoint}"/>
+        <constructor-arg index="0" value="${jira.endpoint}"/>
+        <constructor-arg index="1" value="${jira.username}"/>
+        <constructor-arg index="2" value="${jira.password}"/>
     </bean>
 
     <bean id="noop" class="org.vividus.http.context.NoopHttpContextFactory" />
 
-    <bean class="org.vividus.jira.JiraFacade" />
+    <bean id="jiraFacade" class="org.vividus.jira.JiraFacade" />
```

Comment:
you can use constructor-based dependency injection, and this `id` won't be needed